docs/getting-started-scalar-helm-charts.md

```diff
@@ -53,7 +53,7 @@ First, you need to install the following tools used in this guide.
 
 After the Kubernetes cluster starts, you can try each Scalar Helm Charts on it. Please refer to the following documents for more details.
 
-* [ScalarDB Server](./getting-started-scalardb.md)
+* [[Deprecated] ScalarDB Server](./getting-started-scalardb.md)
```

Comment:
What do you think about moving this link for ScalarDB Server to the bottom of the list since it's deprecated?

Reply:
Sounds good! I will move it to the bottom.
Thank you!

Reply:
I updated it in b1f5040935d3da3e39501a4f68cd0b0d24fd9bcd.